rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -56,6 +54,7 @@ object SystemProcesses {
             ListParWithRandom(Seq(Par.defaultInstance), rand),
             false
           )(MATCH_UNLIMITED_PHLOS)
+          .map(unpackOption(_))
```

Comment:
You can omit `(_)`

Reply:
it didn't work. At least Intellij was complaining.